src/pages/CreateContact.tsx

```diff
@@ -23,27 +23,29 @@ export const validateAddress = async ({
     address?: string;
     profile: Contracts.IProfile;
 }): Promise<ValidateAddressResponse> => {
+    if (!address) {
+        return { isValid: false, network: WalletNetwork.MAINNET };
+    }
+
     try {
-        if (address) {
-            for (const network of [Network.MAINNET, Network.DEVNET]) {
-                try {
-                    await profile.walletFactory().fromAddress({ address, coin: COIN_ID, network });
```

Comment:
adding a card to look at the internals of this method to figure out why the network is seemingly ignored